modules/notes/assets/js/hooks/ui/panel/state-ready/notes-add-panel-menu-item.js

```diff
@@ -13,13 +13,24 @@ export class NotesAddPanelMenuItem extends $e.modules.hookUI.After {
 			icon: 'eicon-commenting-o',
 			title: __( 'Notes', 'elementor' ) + '<i class="elementor-panel-menu-item-title-badge eicon-pro-icon"></i>',
 			callback() {
+				const hasProAndNotConnected = elementor.helpers.hasProAndNotConnected();
+
 				elementor.promotion.showDialog( {
-					headerMessage: __( 'Notes', 'elementor' ),
-					message: __( 'With Notes, teamwork gets even better. Stay in sync with comments, feedback & more on your website.', 'elementor' ),
-					top: '-3',
-					inlineStart: '+10',
-					element: this.$el,
-					actionURL: 'https://go.elementor.com/go-pro-notes/',
+					title: __( 'Notes', 'elementor' ),
+					content: __( 'With Notes, teamwork gets even better. Stay in sync with comments, feedback & more on your website.', 'elementor' ),
+					position: {
+						top: '-3',
```

Comment:
I think it should be:
```suggestion
						blockStart: '-3',
```